src/util/misc.js

```diff
@@ -805,6 +944,13 @@
         translateX: a[4],
         translateY: a[5]
       };
+      Object.defineProperty(options, 'matrix', {
+        value: a,
+        configurable: false,
+        enumerable: false,
+        writable: false
+      });
+      return options;
```

Comment:
i don't think this is good.
Wha happens if i do:

```js
var a = decompose(matrix);
a.translateX = 20;
var b = compose(a);
```

if i hit the shortcut this is wrong.

Reply:
Good point
fb83a7152c8581d67b32caa9d6ba51fccfadb048